src/main/kotlin/com/example/gymi/global/error/exception/BasicException.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.gymi.global.error.exception
+
+import com.example.gymi.global.error.ErrorCode
+
+class BasicException(
+
+        val errorCode: ErrorCode
+): RuntimeException()
```

Comment:
kotlin 공식문서에 상속에서 콜론 앞에는 공백을 추가하라고 나와있어서
```suggestion
) : RuntimeException()
```
이런식으로 수정하면 좋을거 같아요.
https://kotlinlang.org/docs/coding-conventions.html#colon

Reply:
앞으로 작성할 코드에서도 참고하겠습니다 감사합니다!